vcpkg.json

```diff
@@ -2,6 +2,10 @@
   "builtin-baseline": "000d1bda1ffa95a73e0b40334fa4103d6f4d3d48",
   "dependencies": [
     "gtest",
-    "zlib"
+    "zlib",
+    {
+      "name": "sdl2",
+      "version>=": "2.30.1"
```

Comment:
`builtin-baseline` is set to `000d1bda1ffa95a73e0b40334fa4103d6f4d3d48`, but [I don’t think that that commit contains SDL 2 v2.30.1 or any later versions of SDL](https://github.com/microsoft/vcpkg/blob/000d1bda1ffa95a73e0b40334fa4103d6f4d3d48/versions/s-/sdl2.json). I think that either the baseline or the version of SDL needs to be changed.

Reply:
Good catch, updated builtin-baseline version and SDL to 2.30.3

Reply:
See the [override section](https://github.com/DescentDevelopers/Descent3/pull/469/commits/828f903b44d851ffccf47a0409d5ed06ebbc122e#diff-dbbb1b147126744a5eee012901d2b9a29cc478be03677f67825b9b2794f5d283) in #469 - we need an override in order to set a specific single version. Otherwise the `builtin-baseline` and `version>=` (note `>=`) fields just set the _minimum_, not exact, version

Reply:
Where is that override in the link you sent? I only see different features for Linux

Reply:
gah. i didn't look closely enough; i havne't pushed the particular commit i meant to link. will add suggested modifications here instead, that show the change needed.